projects/rpx-xui-translation/src/lib/rpx-translate.pipe.ts

```diff
@@ -8,13 +8,14 @@ import { Replacements, RpxTranslationService } from './rpx-translation.service';
   name: 'rpxTranslate',
   pure: false
 })
-export class RpxTranslatePipe extends AsyncPipe implements PipeTransform {
+export class RpxTranslatePipe implements PipeTransform, OnDestroy {
+  private asyncPipe: AsyncPipe;
 
   constructor(
     private translationService: RpxTranslationService,
-    ref: ChangeDetectorRef
+    private injector: Injector
   ) {
-    super(ref);
+    this.asyncPipe = new AsyncPipe(injector.get(ChangeDetectorRef));
```

Comment:
@MateiHMCTS - query about why this pipe no longer extends the `AsyncPipe` but I think @ronaldmansveld is going to comment on this